src/safe-web/read_only/safe_balances.test.js

```diff
@@ -52,7 +50,7 @@ describe('Safe Balances', () => {
     const newSelectedCurrency = 'EUR'
     await clickSomething(safeBalancesPage.currency_dropdown_btn.selector, gnosisPage, 'css')
     await clickElement(safeBalancesPage.currency_item_label(newSelectedCurrency), gnosisPage)
-    await gnosisPage.waitForTimeout(6000)
+    await gnosisPage.waitForTimeout(4000)
```

Comment:
I had increased this timeout in the last contribution to the repo because running the tests through GitHub was failing to assert the next line with a shorter timeout.

Reply:
It was working fine with 4k, but I don't mind leaving it with 2 more secs.